great_expectations/render/renderer/notebook_renderer.py

```diff
@@ -141,7 +141,8 @@ def add_expectation_cells_from_suite(self, expectations):
 
             for exp in expectations:
                 kwargs_string = self._build_kwargs_string(exp)
-                self.add_code_cell(f"batch.{exp['expectation_type']}({kwargs_string})")
+                meta_args = f", meta={exp.meta}" if exp.meta else ""
```

Comment:
f-strings are not py2 compatible

Reply:
Lol. I managed to sneak them by you before because I knew we didn't have CI covering this branch. 😄 Fixed.